management/guardian.go

```diff
@@ -84,6 +84,13 @@ func (mfm *MultiFactorManager) List(opts ...ListOption) (mf []*MultiFactor, err
 	return
 }
 
+// Deletes an enrollment
```

Comment:
```suggestion
// Delete an enrollment
```

This should get rid of the warning